src/components/App.js

```diff
@@ -17,14 +19,35 @@ class App extends Component {
     const questionsUrl = () => `http://localhost:3001/quizzes`;
```

Comment:
A better name for this might be `quizUrl` since this is actually fetching an entire quiz (with the quiz title and ID), not just the questions.